### PR TITLE
Docs/add wallet creation to faq

### PR DESCRIPTION
**Pull request by carpawell**
Add instruction for New wallet creation
using a binary private key to /docs/faq.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -37,6 +37,31 @@ $ xxd wallets/wallet.key
 
 Later this functionality will be included in `neofs-cli` directly.
 
+### How to create Neo wallet JSON file using a private key?
```
Comment (alexvanin) [inline docs/faq.md:40]: using a NeoFS key file.

File: docs/faq.md [hunk 2]
```diff
@@ -37,6 +37,31 @@ $ xxd wallets/wallet.key
...
+
+You will need `neo-go` and `neofs-cli`.
+
+1. Get the WIF format of the private key
+
+```
+$ neofs-cli util keyer -key ./PATH/TO/YOUR/KEY | grep WIF | awk '{print $NF}' > temp_WIF
```
Comment (alexvanin) [inline docs/faq.md:47]: Let's use key file from the repository as an example, e.g. service/http_gate/node.key

File: docs/faq.md (hunk 1 above)
Comment (carpawell) [reply docs/faq.md]: changed

File: docs/faq.md (hunk 2 above)
Comment (carpawell) [reply docs/faq.md]: added `./services/ir/01.key` as an existing example

**Review by alexvanin — APPROVED** [review]

**Pull request merged**